### DISCUSSION TITLE
[V3] Navigate does not work with browser "back" button

### DISCUSSION THREAD

**Discussion by ricardo-lobo** [livewire/livewire#5805] [7. Bugs (archived)] [answered]
Trying to use the new feature "navigate". It seems to work fine when I click a link. But when I press the browser "go back" button the page does not update.

I tested this with a fresh Laravel fresh install and Livewire 3 beta 2, with Edge and Chrome.

See repo https://github.com/ricardo-lobo/livewire-sandbox

Click the login link for example, press the back button in the browser.

https://github.com/livewire/livewire/assets/21181036/3672b70c-e214-4dc1-b5f6-37983e65c921

**Comment by nicolagianelli** [comment]
I tried it and the problem was the livewire component in the welcome page and the missing @vite(['resources/css/app.css', 'resources/js/app.js']) on the head of the welcome page.
If you add the @vite(['resources/css/app.css', 'resources/js/app.js']) on the head of the welcome page the problem goes away.
Probably it depends on how livewire treats the new head when you go back..

**Reply by ricardo-lobo** [reply]
the @vite is a requirement?

**Comment by PhiloNL** [comment]
I think the issue is caused because the login page doesn't contain any Livewire components and therefore Livewire is no longer injected into the page. Can you try and manually add `@livewireScripts` and `@livewireStyles` to the `resources/views/layouts/guest.blade.php` and disable `auto_asset_inject` in https://github.com/ricardo-lobo/livewire-sandbox/blob/main/config/livewire.php#L104

**Comment by joshhanley** [comment, marked as answer]
I submitted a failing test PR #6262 which has been fixed and merged. So this will be in the next release. 🙂
